api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -570,9 +570,14 @@ enum IntegrationEnrollKind {
   INTEGRATION_ENROLL_KIND_MACHINE_ID_GITLAB = 15;
   INTEGRATION_ENROLL_KIND_MACHINE_ID_JENKINS = 16;
   INTEGRATION_ENROLL_KIND_MACHINE_ID_ANSIBLE = 17;
-  INTEGRATION_ENROLL_KIND_SERVICENOW = 18;
-  INTEGRATION_ENROLL_KIND_ENTRA_ID = 19;
-  INTEGRATION_ENROLL_KIND_DATADOG_INCIDENT_MANAGEMENT = 20;
+  INTEGRATION_ENROLL_KIND_MACHINE_ID_AWS = 18;
+  INTEGRATION_ENROLL_KIND_MACHINE_ID_GCP = 19;
+  INTEGRATION_ENROLL_KIND_MACHINE_ID_AZURE = 20;
+  INTEGRATION_ENROLL_KIND_MACHINE_ID_SPACELIFT = 21;
+  INTEGRATION_ENROLL_KIND_MACHINE_ID_KUBERNETES = 22;
+  INTEGRATION_ENROLL_KIND_ENTRA_ID = 23;
+  INTEGRATION_ENROLL_KIND_DATADOG_INCIDENT_MANAGEMENT = 24;
+  INTEGRATION_ENROLL_KIND_SERVICENOW = 25;
```

Comment:
I think we should just accept that these new Machine ID enums are separated from the older ones, not a big deal really.

```suggestion
  INTEGRATION_ENROLL_KIND_SERVICENOW = 18;
  INTEGRATION_ENROLL_KIND_ENTRA_ID = 19;
  INTEGRATION_ENROLL_KIND_DATADOG_INCIDENT_MANAGEMENT = 20;
  INTEGRATION_ENROLL_KIND_MACHINE_ID_AWS = 21;
  INTEGRATION_ENROLL_KIND_MACHINE_ID_GCP = 22;
  INTEGRATION_ENROLL_KIND_MACHINE_ID_AZURE = 23;
  INTEGRATION_ENROLL_KIND_MACHINE_ID_SPACELIFT = 24;
  INTEGRATION_ENROLL_KIND_MACHINE_ID_KUBERNETES = 25;
```

Reply:
Hmm I see now that the issue is that we're out of sync with prehog proto... I'm not sure off the top of my head what the right thing to do here is.